pylint/pyreverse/diagrams.py

```diff
@@ -121,16 +121,18 @@ def get_relationship(
     def get_attrs(self, node: nodes.ClassDef) -> list[str]:
         """Return visible attributes, possibly with class name."""
         attrs = []
-        properties = [
-            (n, m)
-            for n, m in node.items()
-            if isinstance(m, nodes.FunctionDef) and decorated_with_property(m)
-        ]
-        for node_name, associated_nodes in (
-            list(node.instance_attrs_type.items())
-            + list(node.locals_type.items())
-            + properties
+        properties = {
+            name: attr
+            for name, attr in node.items()
+            if isinstance(attr, nodes.FunctionDef) and decorated_with_property(attr)
+        }
+        for name, attr in list(node.locals_type.items()) + list(
+            node.instance_attrs_type.items()
```

Comment:
Here, `locals_type` and `instance_attrs_type` are mappings from attribute name to possible types (from annotations or inference results), so `for attr_name, attr_type in ...` would be better.